### PR TITLE
Support for PLY format

### PR DESCRIPTION
**Pull request by mmcauliffe**
Allows for importing from and exporting to PLY format through VTK.

I added some tests for export (in the same vein as the current OBJ tests).

### CODE REVIEW

File: menpo3d/io/test/io_export_test.py [hunk 1]
```diff
@@ -28,6 +36,20 @@ def test_export_mesh_obj_textured(mock_open, exists, PILImage, PIL):
...
+@patch('menpo.image.base.PILImage')
+@patch('menpo3d.io.output.base.Path.exists')
+@patch('menpo.io.output.base.Path.open')
+def test_export_mesh_ply_textured(mock_open, exists, PILImage, PIL):
+    PIL.return_value.Image.EXTENSION = {'.jpg': None}
+    exists.return_value = False
+    fake_path = '/fake/fake.ply'
+    MagicMock.name = PropertyMock(return_value=fake_path)
+    m = MagicMock()
+    #type(m).name = fake_path
```
Comment (patricksnape) [inline menpo3d/io/test/io_export_test.py]: You can remove this comment I guess?

**Comment by patricksnape** [comment]
@mmcauliffe Thanks a lot for this! I'll just test with some PLY files now.

**Comment by patricksnape** [comment]
jenkins, test this please

**Comment by patricksnape** [comment]
@mmcauliffe I added a PR [here](https://github.com/mmcauliffe/menpo3d/pull/1) that just makes some minor adjustments and fixes the fact that menpo only really supported file handles rather than paths directly.

**Comment by mmcauliffe** [comment]
Yeah, that looks much better than my initial hack to get it working!

**Comment by patricksnape** [comment]
Thanks @mmcauliffe, sorry this took so long to get in!

**Pull request merged**